File: examples/qr/generate/go/generateAQrCode.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/qr"
)

// Generate a QR code with a specific text and size
func GenerateAqrCode() {
	qrService := qr.NewQrService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := qrService.Generate(&qr.GenerateRequest{
		Size: 300,
		Text: "https://m3o.com/qr",
	})
	fmt.Println(rsp, err)
}

File: examples/js/qr/generate/generateAQrCode.js
const { QrService } = require("m3o/qr");

// Generate a QR code with a specific text and size
async function generateAqrCode() {
  let qrService = new QrService(process.env.MICRO_API_TOKEN);
  let rsp = await qrService.generate({
    size: 300,
    text: "https://m3o.com/qr",
  });
  console.log(rsp);
}

generateAqrCode();
